src/helper/class-ee-site.php

```diff
@@ -758,6 +758,19 @@ protected function shut_down_function() {
 		}
 	}
 
+	/*
+	 *  Check site count for maximum 27 sites.
+	 */
+	protected function check_site_count() {
+		$sites = Site::all();
+
+		if ( 27 > count( $sites) ) {
```

Comment:
Did you run phpcbf?
There should be space between `$sites` and `)` as mentioned in previous changes too.